api-reference/10 UI Widgets/dxDataGrid/1 Configuration/remoteOperations/summary.md

```diff
@@ -8,4 +8,4 @@ default: false
 Specifies whether or not [summaries](/concepts/05%20Widgets/DataGrid/65%20Summaries/10%20Total%20Summary '/Documentation/Guide/Widgets/DataGrid/Summaries/Total_Summary/') calculation must be performed on the server.
```

Comment:
Can we be more direct (and avoid the word "perform"):
Specifies whether or not summaries are calculated on the server-side.